examples/recognizer/embedded_recognize_once_async_from_file.rs

```diff
@@ -0,0 +1,33 @@
+use super::helpers;
+use cognitive_services_speech_sdk_rs::audio::AudioConfig;
+use cognitive_services_speech_sdk_rs::speech::{EmbeddedSpeechConfig, SpeechRecognizer};
+use log::*;
+use std::env;
+
+#[allow(dead_code)]
+pub async fn run_example() {
+    info!("-----------------------------------------------");
+    info!("running embedded recognize_from_file example...");
+    info!("-----------------------------------------------");
+
+    let filename = helpers::get_sample_file("hello_rust.wav");
+    let audio_config = AudioConfig::from_wav_file_input(&filename).unwrap();
+
+    let mut speech_config =
+        EmbeddedSpeechConfig::from_path(env::var("ModelPath").unwrap()).unwrap();
```

Comment:
can we provide in this example more info about embedded model? how/where to get it so that this example can be tested really easily without too much exploration etc.